synapse/tests/test_model_infotech.py

```diff
@@ -1439,7 +1439,9 @@ async def test_infotech_repo(self):
                     'repo': repo,
                     'title': 'a fancy new release',
                     'desc': 'Gonna be a big release friday',
-                    'created': 0,
+                    'created': 1,
```

Comment:
Looks like you might be missing some property checks for some of the assignments in here :)